src/Components/Footer/Footer.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import logoFooter from '../../assets/images/logoFooter.png';
+import dribbble from '../../assets/images/Dribbble.png';
+import instagram from '../../assets/images/Instagram.png';
+import twitter from '../../assets/images/Twitter.png';
+import youtube from '../../assets/images/Youtube.png';
+
+function Header() {
+  return (
+    <div
+      style={{ backgroundColor: '#4FAA4D' }}
```

Comment:
I guess u can add a class "bg-primary" instead of applying a style

Reply:
Noted, I changed it and pushed it again 
And I added the © to the footer too